doc-surrealdb_versioned_docs/version-2.x/surrealql/statements/define/access/record.mdx

```diff
@@ -172,9 +168,11 @@ ALGORITHM HS512 KEY "secret";
 
 The authenticate clause can be used to change the record identifier returned by the `SIGNIN` and `SIGNUP` clauses or replace the identifier provided in the token when authenticating `WITH JWT`. This clause can also be used to log or stop authentication attempts from record users, as it is always executed across signin, signup and token authentication. The clause always expects a record identifier to be returned and will otherwise fail.
 
+Unlike the [`PERMISSIONS`](/docs/surrealdb/2.x/surrealql/statements/define/table#defining-permissions) clause, the `AUTHENTICATE` clause is executed only at the time of authentication, resulting in increased performance for queries that only need to be validated at that point. On the other hand, permissions queries are executed in every query and for each record, ensuring that any authorization conditions are verified at the time of the query. The `AUTHENTICATE` clause is a good fit for validating specific conditions that are not expected to change during the lifetime of the session such as the presence of any required token claims.
+
 #### Example: External authentication providers
 
-Replacing the record identifier that will be used to establish the session is especially useful in scenarios where the token used to authenticate the session does not contain one. This is common when using an external authentication provider, which may only have knowledge of generic user identifiers such as an email address or UUID.
+Replacing the record identifier that will be used to establish the session is specially useful in scenarios where the token used to authenticate the session does not contain one. This is common when using an external authentication provider, which may only have knowledge of generic user identifiers such as an email address or UUID.
```

Comment:
In the context of this sentence it should be   especially as it relates to “a particular purpose”

Reply:
Damn, I thought that I got it wrong the first time :joy: 
Changed in 7a763c5670b467f1a477cf1d62964c83eb57fc79.

Reply:
Ah not at all!